src/main/java/io/anserini/collection/CoreCollection.java

```diff
@@ -123,6 +125,13 @@ public Document(JsonNode json) {
                   "doi:" + json.get("doi").asText();
         } else if ("abstract".equals(e.getKey())) {
           this.contents = json.get("title").asText() + "\n" + json.get("abstract").asText();
+        } else if ("topics".equals(e.getKey())) {
+          ArrayNode arrayField = (ArrayNode) e.getValue();
+          StringJoiner sj = new StringJoiner(" ");
```

Comment:
I would suggest no use "space" to join the array, since "topics" consist of many phrases.

Reply:
Updated to `::` as @lintool suggested, Lucene should ignore the punctuation when indexing so this should not be an issue right?